pkg/services/ngalert/notifier/cluster.go

```diff
@@ -0,0 +1,15 @@
+package notifier
+
+import (
+	"time"
+
+	"github.com/prometheus/alertmanager/cluster"
+)
+
+// These constants are used and set via the settings package.
+const (
+	DefaultClusterAddr      = "0.0.0.0:9094"
```

Comment:
I think I would prefer we listen on localhost `127.0.0.1` as a default rather than all interfaces, then users can set the address as required?

Reply:
Discussed this offline, we went with `0.0.0.0` as this is what the Alertmanager uses by default.